docs/dealing-with-large-issue-metrics.md

```diff
@@ -0,0 +1,17 @@
+# Dealing with large issue metrics Markdown files
+
+When working with lots of issues/pull requests/discussion results, the resulting issue_metrics.md  file can become very large. This can cause the GitHub API to return an error when trying to create an issue with the contents of the file.
```

Comment:
```suggestion
When working with lots of issues/pull requests/discussion results, the resulting issue_metrics.md file can become very large. This can cause the GitHub API to return an error when trying to create an issue with the contents of the file.
```